index.js

```diff
@@ -10,9 +10,39 @@ function parseGitUri(uri) {
     uri.match(/git@github.com:(.*)\/(.*)\.git/);
 }
 
+function verifyChangelog() {
+  var changelog = exec('git diff HEAD~1');
+  var lines = changelog.split('\n');
+  var isOneLine = (lines.length === 1);
+  var deletions = 0;
+  var containsErrorMessage = false;
+  lines.filter(function (line) {
+    if (/^-/.test(line)) deletions++;
+    if (line.indexOf('Make a POST first') >= 0) containsErrorMessage = true;
+  });
+  if (isOneLine) {
+    revertChanges();
+    console.error('Changelog diff should be more than 1 line long');
+    process.exit(3);
+  }
+  if (deletions > 10) {
```

Comment:
Well, should there be any deletions at all? It seems like the if there would be a difference in the changelog, it would only be in the form of additions (unless I am missing something).